src/services/pawapay/pawapay.ts

```
@@ -0,0 +1,450 @@
+    } catch (error) {
+      console.error('Error resending payout callback:', error);
+      throw error;
+    }
+  }
+
+  // Validate a callback signature
+  validateCallbackSignature(
+    headers: Record<string, string>,
+    body: any,
+    publicKey: string
+  ): boolean {
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)